common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -240,8 +261,14 @@ def create_event(self, knowledge_node_uid, student_uid, event_type, **data):
         payload.update(data)
 
         # Send request
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        event = json.loads(response.content)
+        try:
+            response = requests.post(url, headers=self.request_headers, data=payload)
+            event = json.loads(response.content)
+        except requests.ConnectionError:
+            event = {}
+            log.warning("Could not create event of type {event_type} for "
+                        "unit identified by knowledge_node_uid {knowledge_node_uid} "
+                        "and student identified by student_uid {student_uid}.".format(event_type=event_type, student_uid=student_uid, knowledge_node_uid=knowledge_node_uid))
```

Comment:
@macornwell This line is still too long. [Line length should not exceed 120 chars.](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization)